core/include/clist.h

```diff
@@ -102,6 +104,20 @@ extern "C" {
  */
 typedef list_node_t clist_node_t;
 
+/**
+ * @brief Checks if *list is empty
+ *
+ * @note Complexity: O(1)
+ *
+ * @param[in]   list        Pointer to clist
+ *
+ * @returns     true if list has any elements, false if not
```

Comment:
lol, I reversed the relevant documentation, will fix

Reply:
Just squash directly